tests/acceptance/characters/control/test_web_control_blockchain.py

```diff
@@ -123,7 +123,7 @@ def test_alice_character_control_revoke(alice_web_controller_test_client, blockc
         'label': 'test-revoke',
         'm': 2,
         'n': 3,
-        'expiration': (maya.now() + datetime.timedelta(days=3)).iso8601(),
+        'expiration': (maya.now() + datetime.timedelta(days=35)).iso8601(),
```

Comment:
Tests for policies are not perfect because they use real `now` for time instead of blockchain `now`, and we time travel in tests a lot. So here should just some big value that policy still be active in test moment